### PR TITLE
Hack at wheel to override tag

### PR DESCRIPTION
**Pull request by auscompgeek**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+s = Setup()
+s.prepare()
+
+try:
+    from wheel.bdist_wheel import bdist_wheel as _bdist_wheel
+except ImportError:
+    pass
+else:
+
+    class bdist_wheel(_bdist_wheel):
```
Comment (virtuald) [inline setup.py:15]: This should be added to robotpy-build instead?

Comment (auscompgeek) [reply setup.py:15]: Probably, yeah. This was more a PoC than anything I expected to actually be merged. Would be good to get rid of `setup.py` everywhere anyway.

I'm not too sure how to best integrate this into robotpy-build proper though.

**Comment by auscompgeek** [comment]
We should probably figure out what to do about this. Is this hack appropriate to include in robotpy-build? It certainly feels like a hack on top of another hack to me...

**Comment by virtuald** [comment]
If we're going to do it, definitely belongs in robotpy-build and not here. I'm not terribly opposed to it.

**Comment by auscompgeek** [comment]
OBE with the extension module for signal handling

**Pull request closed without merging**